react-accordion-component/my-app/src/Accordion Components/Accordion.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+
+type AccordionProps = {
```

Comment:
Since this type isn't exported, I personally prefer to just call it Props. Less typing.

Reply:
Removed this and imported Topics instead